lib/issues.test.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+var jQuery = require("jquery");
+var refute = require("@sinonjs/referee").refute;
+
+var samsam = require("./samsam");
+
+describe("issues", function() {
+    describe("issue #71", function() {
+        before(function() {
+            if (typeof window === "undefined") {
+                this.skip();
+            }
+        });
+
+        context("when comparing jQuery objects", function() {
+            it("should not throw", function() {
+                refute.exception(function() {
+                    samsam.deepEqual(jQuery("#test"), jQuery("#test"));
```

Comment:
Maybe the query object would compare differently if it would wrap an actual DOM object, e.g. `body`? I think this creates an object wrapping `null` because the element doesn’t exist.

Reply:
Sure, I can do that

Reply:
Ha! I think that `samsam.deepEqual(jQuery("body"), jQuery("#body"))` should fail everywhere because these should be different.

Reply:
It should return `false`, but it shouldn't throw, as `samsam` is not an assertion library

Reply:
Yes of cause, you're right. Thanks for adding tests anyway 👍